src/js-form-validator.ts

```diff
@@ -313,10 +313,8 @@ export class FormValidator {
       case 'phone':
         if (!allowEmpty && !value.length) return -1;
         let vals = value.split('x');
-        for (let i = 0, l = vals.length; i < l; i++)
-          vals[i] = vals[i].replace(/\D/g, '');
         value = vals.join('x').replace(/^1/, ''); // remove leading 1 if added, only for US
-        status = /^[0-9x]{7,16}$/.test(value) ? 1 : 0;
+        status = /^[0-9\-\+\s\(\)]{7,16}$/.test(value) ? 1 : 0;
```

Comment:
I don't think we should be validating this way. We only care about the numbers, not any other formats, which is why the original formatting stripped it all out. This allows us not to care if a plus sign is attached as the first character or not, or if they are or are not using dashes.

So, this begs the question, do we care about presentation?